### PR TITLE
9283 poly

### PR DESCRIPTION
**Pull request by cortlandstarrett**

### CODE REVIEW

File: model/maslin/models/maslin/m2x/ooapopulation/ooapopulation.xtuml [hunk 1]
```diff
@@ -6461,12 +6461,17 @@ INSERT INTO O_TFR
 // not really a state but is "Ignore" or "Cannot_Happen"
 
 select any fromState related by sm_sm->SM_STATE[R501] where (selected.Name == param.startState);
+// Give preference to finding an inherited event (by overwriting a local find).
+// This allows a dubious possibility of having a local event with the
+// same name as an event in the supertype.
```
Comment (leviathan747) [inline model/maslin/models/maslin/m2x/ooapopulation/ooapopulation.xtuml]: Is there ever a case where param.object != "" and there is not a supertype poly? Would that be the case if the modeler fully qualified the event with its own containing domain name?

File: doc/notes/9283_poly_int.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+3. Background
+-------------
+BridgePoint supports abstract polymorphic events as defined in [2] and [4].
+iUML supports concrete polymorphic events as described in [3].  This work
+moves to a strategy of concrete polymorphic events using a name-based
+strategy.  Inherited events are named with the MASL dotted syntax
+(super.eventName).  The presence of the dot in the name is the key to
+distinguishing events inherited from supertypes from events locally
+defined within the (present) subtype.
```
Comment (leviathan747) [inline doc/notes/9283_poly_int.md]: Define some clear terms and use them consistently throughout the note. "Concrete polys" v. "Abstract polys" is the main one. They can read the literature but a concise definition would be helpful for future readers of this note. The biggest thing was it was confusing whether you were talking about actual polys (SM_PEVT) and concrete events (SM_SEVT) being used as polys

File: model/maslin/models/maslin/m2x/ooapopulation/ooapopulation.xtuml (hunk 1 above)
Comment (cortlandstarrett) [inline model/maslin/models/maslin/m2x/ooapopulation/ooapopulation.xtuml]: This is a good question.  I will ask Tristan on the issue in Redmine.

File: doc/notes/9283_poly_int.md (hunk 2 above)
Comment (cortlandstarrett) [inline doc/notes/9283_poly_int.md]: I will look at this and see if an explicit definition would help the note.

**Pull request merged**